Cabal-tests/tests/ParserTests/regressions/all-upper-bound.check

```diff
@@ -1,6 +1,6 @@
 These packages miss upper bounds:
+  - somelib
```

Comment:
This change is due to ditching the `nub`s that you mention in the blog post, right?

Reply:
No, the old check reordered the dependencies by name, a thing I do not do to because it is easier for the user to fix them in the order written in the `.cabal` file.

Reply:
See [here](https://github.com/haskell/cabal/blob/90c3f6b8b62334fefe166e825d0ec16b6808cbe2/Cabal-tests/tests/ParserTests/regressions/all-upper-bound.cabal), `somelib` (the first offending library) comes *before* `alpha`, etc.